UI/style-guest.css

```diff
@@ -0,0 +1,349 @@
+/* html, body, body div, span, object, iframe, h1, h2, h3, h4, h5, h6, p, blockquote, pre, abbr, address, cite, code, del, dfn, em, img, ins, kbd, q, samp, small, strong, sub, sup, var, b, i, dl, dt, dd, ol, ul, li, fieldset, form, label, legend, table, caption, tbody, tfoot, thead, tr, th, td, article, aside, figure, footer, header, menu, nav, section, time, mark, audio, video, details, summary {
```

Comment:
Instead of having two copies of the same stylesheets to load on different pages you could extract all common styles to separate css file and link it to all pages when they're required. All the page-specific stylesheets may be moved to %page%.css file and imported where required.
And did have really created all the style rules below by yourself?